### ISSUE TITLE
CLI Data triangulation: Network, Application, PSAT

### ISSUE THREAD

**Issue by maitreyie-chavan** [GoogleChromeLabs/ps-analysis-tool#628]
**Description:**

To ensure consistency between extension and CLI results for all cookies across frames, we must gather and monitor data from the Network and other panels to accurately associate cookies with their respective frames. This process will eliminate the need for the 'unmapped frames' section in the CLI report

**Comment by ayushnirwal** [comment]
'Unknown Frames' come from `Page.getCookies` query resulting in cookie data without any frame-related data. Therefore we had to attribute those cookies to 'Unknown Frames'. The usage of `Page.getCookies` was necessary to add data about the cookies that were missed in Network events.
This was solved in PR #704 by relying on CDP and using [Target.setAutoAttach](https://chromedevtools.github.io/devtools-protocol/tot/Target/#method-setAutoAttach) to flatten the target tree. This eliminates the problem of missing network events when network events of the main frame are listened to.
There is no need to scrape data from the network and application panel and triangulate them to get correct cookie data.
